os/src-jvm/ProcessOps.scala

```diff
@@ -117,7 +119,6 @@ case class proc(command: Shellable*) {
 
     builder.directory(Option(cwd).getOrElse(os.pwd).toIO)
 
-    val commandChunks = command.flatMap(_.value)
```

Comment:
You moved the `commandChunks` upwards and made it a `def`, to make it accessible. `commandChunks` is used twice here, so maybe, we still want to hold it in a local `val`?

Reply:
Well, it was as val in the initial version of this PR. See https://github.com/com-lihaoyi/os-lib/pull/100#discussion_r811763520 :)

Reply:
I know. But in a different position. Here, it's local and private, whereas initially you create a public val which hold redundant data. Anyway. I don't think it will hurt as-is. I'm just pointing it out. Nitpicking it is, sure, but what else can I do, if nobody else is picking up this PR.

Reply:
Made `commandChunks` a `val` again.

Reply:
Made `commandChunks` a `val` again.